src/components/app/DetailsContainer.css

```diff
@@ -0,0 +1,27 @@
+.DetailsContainer .layout-pane > * {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+}
+
+.DetailsContainer .layout-pane:not(.layout-pane-primary) {
+  max-width: 250px;
+}
+
+/* overriding defaults from splitter-layout */
+.DetailsContainer {
+  /* SplitterLayout injects position: absolute, this conflicts with our using
+   * Flex Layout. */
+  position: unset;
+}
+
+.DetailsContainer .layout-splitter {
```

Comment:
As we have `splitter-layout` for the sidebar and `layout-splitter` for separating the map pannel from the details, this might be confusing. Maybe something like `layout-splitter-secondarily` or `sidebar-splitter` would be more clear?

Reply:
Oh, I seeeee :D